modules/guardduty/variables.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_version = "~> 0.11.0"
+  backend          "local"          {}
+}
+
+provider "aws" {
```

Comment:
How does this module exec into an account to enable guard duty?

You might need to assume the roles you created in your other [PR](https://github.com/ministryofjustice/analytical-platform-iam/blob/bef6a1960c4eb46cf89470272c9de7f6502d489d/modules/role/main.tf#L1-L8)?

Example [here](https://github.com/ministryofjustice/analytical-platform-iam/blob/bef6a1960c4eb46cf89470272c9de7f6502d489d/modules/role/main.tf#L1-L8)

Reply:
Oh thanks! yeah I see what you mean, let me fix that